### PR TITLE
[NET-3032] Add documentation for Enhanced Cloud Resolution in NPM product

### PR DESCRIPTION
**Pull request by stb2132**
### What does this PR do?
This updates documentation for the NPM product to include details around the enhanced resolution possible with cloud resource crawls. 

The PR also moves the description of this installation outside of the agent installation documentation, since it is a separate setup process from installing the agent.

### CODE REVIEW

File: content/en/network_monitoring/performance/network_page.md [hunk 1]
```diff
@@ -123,7 +123,56 @@ NPM automatically maps
 - network calls to S3 (which can broken down by `s3_bucket`), RDS (which can be broken down by `rds_instance_type`), Kinesis, ELB, Elasticache, and other [AWS services][3].
 - API calls to AppEngine, Google DNS, Gmail, and other [GCP services][4].
 
-To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by the [`domain` tag](#domain-resolution).
+To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by the [`domain` tag](#domain-resolution). Or, see the section below for cloud service resolution.
+
+### Cloud service enhanced resolution
+If you have [setup][1] enhanced resolution for AWS or Azure, can filter and group network traffic with resources collected from these cloud providers. Depending on the cloud provider and resource, you have different sets of tags available to query with. Azure loadbalancers currently only have user-defined tags. For AWS, Datadog applies the tags defined below in addition to the user-defined tags.
```
Comment (leeavital) [inline content/en/network_monitoring/performance/network_page.md:129]:
```suggestion
If you have [setup][1] enhanced resolution for AWS or Azure, NPM can filter and group network traffic with several resources collected from these cloud providers. Depending on the cloud provider and resource, you have different sets of tags available to query with. Azure loadbalancers only have user-defined tags. For AWS, Datadog applies the tags defined below in addition to the user-defined tags.
```

If we merge this with the Azure caveat, let's add a reminder on the ticket to fix it

**Review by leeavital — APPROVED** [review]

**Review by heyronhay — APPROVED** [review]

File: content/en/network_monitoring/performance/network_page.md [hunk 2]
```diff
@@ -123,7 +123,56 @@ NPM automatically maps
...
+ {{% tab "VPC Internet Gateway" %}}
+ - gateway_id
+ - gateway_type
+ - aws_internet_gateway_id
+ - aws_account
+ - region
+ - custom (user) tags applied to VPC Internet Gateways
+ {{% /tab %}}
+
+{{% tab "VPC Internet Endpoint" %}}
```
Comment (hmahmood) [inline content/en/network_monitoring/performance/network_page.md:164]: VPC Internet Endpoint? Isn't this just "VPC Endpoint"?

File: content/en/network_monitoring/performance/network_page.md [hunk 3]
```diff
@@ -123,7 +123,56 @@ NPM automatically maps
 - network calls to S3 (which can broken down by `s3_bucket`), RDS (which can be broken down by `rds_instance_type`), Kinesis, ELB, Elasticache, and other [AWS services][3].
 - API calls to AppEngine, Google DNS, Gmail, and other [Google Cloud services][4].
 
-To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by the [`domain` tag](#domain-resolution).
+To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by the [`domain` tag](#domain-resolution). Or, see the section below for cloud service resolution.
+
+### Cloud service enhanced resolution
+If you have [setup][1] enhanced resolution for AWS or Azure, NPM can filter and group network traffic with several resources collected from these cloud providers. Depending on the cloud provider and resource, you have different sets of tags available to query with. Azure loadbalancers only have user-defined tags. For AWS, Datadog applies the tags defined below in addition to the user-defined tags.
```
Comment (estherk15) [inline content/en/network_monitoring/performance/network_page.md:129]:
```suggestion
If you have [setup][9] enhanced resolution for AWS or Azure, NPM can filter and group network traffic with several resources collected from these cloud providers. Depending on the cloud provider and resource, you have different sets of tags available to query with. Azure loadbalancers only have user-defined tags. For AWS, Datadog applies the tags defined below in addition to the user-defined tags.
```

File: content/en/network_monitoring/performance/network_page.md [hunk 4]
```diff
@@ -123,7 +123,56 @@ NPM automatically maps
...
+ - gateway_type
+ - aws_vpc_endpoint_id
+ - custom (user) tags applied to VPC Internet Endpoints
+ {{% /tab %}}
+
+
+
+ {{< /tabs >}}
+
+ [1]: /network_monitoring/performance/setup/#enhanced-resolution
```
Comment (estherk15) [inline content/en/network_monitoring/performance/network_page.md:175]: Can you move this to the bottom of the page around ln 277

```suggestion
 [9]: /network_monitoring/performance/setup/#enhanced-resolution
```

File: content/en/network_monitoring/performance/setup.md [hunk 5]
```diff
@@ -450,6 +440,21 @@ To set up on AWS ECS, see the [AWS ECS][1] documentation page.
 {{% /tab %}}
 {{< /tabs >}}
 
+### Enhanced Resolution
```
Comment (estherk15) [inline content/en/network_monitoring/performance/setup.md:443]: Can you move the site-region partial above the title? When I toggle to AP1 or US1-FED, the whole section should be hidden.

```suggestion
{{< site-region region="us,us3,us5,eu" >}}
### Enhanced Resolution
```

File: content/en/network_monitoring/performance/setup.md [hunk 6]
```diff
@@ -450,6 +440,21 @@ To set up on AWS ECS, see the [AWS ECS][1] documentation page.
...
+{{< site-region region="us,us3,us5,eu" >}}
```
Comment (estherk15) [inline content/en/network_monitoring/performance/setup.md:444]:
```suggestion
```

**Review by estherk15 — CHANGES_REQUESTED** [review]

File: content/en/network_monitoring/performance/network_page.md (hunk 4 above)
Comment (stb2132) [reply content/en/network_monitoring/performance/network_page.md]: Did this manually, going to resolve here.

**Review by estherk15 — APPROVED** [review]

**Pull request merged**